api-examples/pretrain-transformer-lm.py

```diff
@@ -526,10 +527,25 @@ def train():
     start_epoch = 0
     if args.restart_from:
         model.load_state_dict(torch.load(args.restart_from))
```

Comment:
Speaking of legacy ckpts, when i load the ones created by v1 and converted to v2, this line causes error because those ckpts don't have the overall layernorm. We probably want `strict=False` and a warning if there's mismatch of keys.

Reply:
Lets handle this in a separate PR